packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -28,13 +28,16 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
     int n_values = 50000;
     int n_queries = 20000;
     int n_neighbors = 10;
+    int buffer_size = 5;
```

Comment:
Should that one be `0`?

Reply:
Yes I think so. Since we have no clue of the size, there is no point in putting a value.